Rakefile

```diff
@@ -30,8 +35,16 @@ task spec: [:spec_no_clients,
            ]
 
 task :build do
-  system 'rm ./cisco_os_shim*.gem'
-  Dir['*.gemspec'].each { |gemspec| system "gem build #{gemspec}" }
+  system 'rm -f ./cisco_os_shim*.gem'
```

Comment:
Will dump a complaint if the files don't exist - maybe check for existence first?